README.md

```diff
@@ -1,2 +1,54 @@
-# weather-app
-this is a simple application for getting the weather based on your city built with javaScript
+# Weather forecast app
+
+> This repo contains an implementation of a weather forecast app using the open wethr api and sync functions.
+
+![screenshot](./shoot.png)
+
+## Built With
+
+- javaScript,
+- Html,
+- css,
+- boostrap,
+- webpack
+- open weather APi,
+
+## Live Demo
+
+[Live Demo Link](https://rawcdn.githack.com/Tresor11/weather-app/3cc4e760f3642dca31b1fa2050b495aa0d8d7483/dist/index.html)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+make sure you have node and npm istalled on you local computer
+### clone the repo by typing
+```git clone https://github.com/Tresor11/weather-app.git```
+### Install
+run 
+```npm install```
+to get the dependencies
+open your local server and you shpu;d be good to go
+
```

Comment:
after `npm install` i think, one has to run webpack too and open `index.html` file
- add instructions for that